GCRCatalogs/dc2_object.py

```diff
@@ -678,10 +655,23 @@ def _subclass_init(self, **kwargs):
         super()._subclass_init(**kwargs)
 
     def _detect_available_bands(self):
-        return [col.rpartition('_')[0] for col in self._columns if col.endswith('_FLUXMAG0')]
+        """
+        This method should return available bands in the catalog file.
+        For object catalog files generated by DC2-production/scripts/make_object_catalog.py,
+        columns `<band>_FLUXMAG0` would exist.
+        For DPDD-only object catalog files generated by DC2-production/scripts/write_gcr_to_parquet.py,
+        columns `psFlux_<band>` would exist.
+        This function first check `<band>_FLUXMAG0`, and if none exist, check `psFlux_<band>`.
+        If neither exists, returns an empty list.
+        Note that band name may contain underscroll.
```

Comment:
""" suggestion
This function first checks for `<band>_FLUXMAG0` columns, and if none exists, then checks for `psFlux_<band>`.
If neither exists, it returns an empty list.
Note that band name may contain underscores
"""

Reply:
Thanks for the suggestion! Fixed. 

(BTW, use ``` to open and close suggestion box will enable GitHub's suggestion mode.)